tools/systemd/generate-pgroonga-primary-maintainer-service.sh

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env bash
+
+set -xue
+
+pgroonga_primary_maintainer_command=$(which pgroonga-primary-maintainer.sh || :)
+threshold="1G"
+psql_command=""
+environments=()
+on_failure_service=""
+
+function usage () {
+  cat <<USAGE
+Options:
+-s, --pgroonga-primary-maintainer-command:
```

Comment:
Why `-s`?

Reply:
I couldn't think of a good one, so I chose `s` for `script`.